addon.json

```diff
@@ -0,0 +1,74 @@
+{
+    "identifier": "com.seeq.addons.udf_ui",
+    "name": "User Defined Functions Formula Editor",
+    "description": "A Seeq add-on tool for managing user-defined formula functions.",
+    "version": "1.0.0",
+    "license": "Seeq",
+    "icon": "fa fa-edit",
+    "tags": {
+        "maintainer": "Seeq"
+    },
+    "previews": [],
+    "elements": [
+        {
+            "name": "User Defined Functions Formula Editor",
+            "identifier": "com.seeq.addons.udf_ui.datalabfunctions",
+            "known_aliases": ["User Defined Formula Functions Editor", "UDF UI", "User Defined Functions","UDF Editor"],
+            "description":"A Seeq add-on tool for managing user-defined formula functions.",
+            "type": "AddOnTool",
+            "path": "data-lab-functions",
```

Comment:
[nit] Suggest `add-on-tool` throughout as a more descriptive folder name

Reply:
fixed in the next push!